ops/ccs-ops-misc/synthetic-data/scripts/test_synthetic_file.py

```diff
@@ -0,0 +1,44 @@
+import sys
+
+def validate_file(filename):
+    '''Validates synthea rif csv files and ensures that the file is valid for loading
+    into the pipeline. This is primarily designed to check synthea files for minor
+    errors post-generation. Validations can be added here as needed.
+    '''
+
+    passed = True
+    
+    with open(filename) as infile:
+        lineCount = 0
+        expectedColumns = 0
+        for line in infile:
+            multiInsertCount = line.count("INSERT")
+            if multiInsertCount > 1:
+                print("Multiple INSERTs on line " + str(lineCount))
+                passed = False
+            multiUpdateCount = line.count("UPDATE")
+            if multiUpdateCount > 1:
+                print("Multiple UPDATES on line " + str(lineCount))
+                passed = False
+            doubleSpaceCount = line.count("  ")
+            if doubleSpaceCount > 0:
+                print("Multiple spaces on line " + str(lineCount))
+                passed = False
+            columnCount = line.count("|")
+            if expectedColumns == 0:
+                expectedColumns = columnCount
+            if columnCount != expectedColumns:
+                print("Bad column count on line " + str(lineCount))
+                passed = False
+            lineCount = lineCount + 1
+            
+    if passed == False:
+        print("Validation failed")
+    else:
+        print("Validation passed")
```

Comment:
Something like the following might be good. Alternatively, adjust to use a `if passed is False` for singleton comparison:
```suggestion
    if passed:
        print("Validation passed")
    else:
        print("Validation failed")
```